internal/boxcli/secrets.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package boxcli
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"go.jetpack.io/devbox/internal/devbox"
+	"go.jetpack.io/devbox/internal/devbox/devopt"
+	"go.jetpack.io/envsec/pkg/envsec"
+)
+
+type secretsFlags struct {
+	config configFlags
+}
+
+type secretsInitCmdFlags struct {
+	force bool
+}
+
+type secretsListFlags struct {
+	show   bool
+	format string
+}
+
+func secretsCmd() *cobra.Command {
+	flags := &secretsFlags{}
+	cmd := &cobra.Command{
+		Use:     "secrets",
+		Aliases: []string{"envsec"},
+		Short:   "Interact with devbox secrets.",
```

Comment:
Instead of `...devbox secrets` should it be `Interact with secrets in jetpack cloud` ?